### PR TITLE
Unstake button always disabled

### PR DESCRIPTION
**Pull request by RanCohenn**

### CODE REVIEW

File: src/services/web3/governance/governance.ts [hunk 1]
```diff
@@ -82,5 +82,5 @@ export const getUnstakeTimer = async (user: string) => {
   );
   const locks = await govContract.voteLocks(user);
   const time = Number(locks) * 1000;
-  return time < 0 ? 0 : time;
+  if (time > 0) return time;
```
Comment (zavelevsky) [inline src/services/web3/governance/governance.ts:85]: please be explicit about what you return if time <= 0

Comment (RanCohenn) [reply src/services/web3/governance/governance.ts:85]: undefined is used in other places, you preffer -1?

Comment (zavelevsky) [reply src/services/web3/governance/governance.ts:85]: it's ok to return undefined, but please explicitly write `return undefined`

**Review by zavelevsky — APPROVED** [review]
LGTM

**Pull request merged**